Src/Geneva.c

```diff
@@ -93,9 +87,12 @@ void geneva_Callback(){
 		}
 	if(geneva_idle != geneva_state){
 		float state = geneva_Encodervalue();
-		int PIDoutput = singlePID(Geneva_pid.ref, state, Geneva_pid.K_terms, Geneva_pid.timestep);
-		PIDoutput = ClipFloat(PIDoutput, -4 * Geneva_pid.max_pwm, 4 * Geneva_pid.max_pwm); //Limit the PID output, legacy, not sure if necessary
-		pid_SetOutput(PIDoutput, &Geneva_pid); //send signal to the motor
+		float err = Geneva_pid.ref- state;
+		if(abs(err)>GENEVA_MAX_ALLOWED_OFFSET*0.5){ //prevents the integrate control from oscillating around zero, and heating the driver
```

Comment:
Why the 0.5 multiplier?

Reply:
GENEVA_MAX_ALLOWED_OFFSET is like 162, in the original code I wanted 75, then we settled on 5 as a definite safe number, then I didn't know what a good number would be anymore

Reply:
Maybe you should change GENEVA_MAX_ALLOWED_OFFSET, I think that makes more sense